helm_factory/chart_info.py

```diff
@@ -0,0 +1,43 @@
+from typing import List, Optional
+
+from helm_factory.kubernetes_objects.base_objects import HelmYaml
+
+
+class ChartInfo(HelmYaml):
+    """
+    Abstraction over chart.yaml
+    Main builder object. Accepts kubernetes objects and generates the helm chart
+    structure. Can also perform the installation onto the server
+    """
+
+    def __init__(
+        self,
+        api_version: str,
+        name: str,
+        version: str,
+        kube_version: str = "",
+        description: str = "",
+        type: str = "",
+        key_words: Optional[List[str]] = None,
+        home: str = "",
+        sources: Optional[List[str]] = None,
+        **kwargs
+    ):
+
```

Comment:
Please change default values to optional for consistency across framework

Reply:
thank you